src/scripts/settings.js

```diff
@@ -32,6 +32,14 @@ document.querySelector('#close-review-prompt').addEventListener('click', dismiss
   link.addEventListener('click', changeActiveTab);
 });
 
+const isNotTogglApp = (val) => {
+  if (!val) {
+    return true;
+  }
+
+  return (!!val.match(/toggl\./)) === false;
```

Comment:
Also, I find this `!!` and `=== false` really detached from the semantics of what the code is doing, regardless of the fact that the output is correct.

I'd propose following the semantics of what the algorithm is actually doing, which is `whether there are any matches`:

`val.match(/toggl\./))?.length > 0`

Reply:
The canonical way for expressing no matches is `null` ☝️:

```
val.match(/toggl\./) !== null
```